test/test_nonhydrostatic_models.jl

```diff
@@ -95,13 +101,13 @@ using Oceananigans.Grids: required_halo_size_x, required_halo_size_y, required_h
     @testset "Model construction with single tracer and nothing tracer" begin
         @info "  Testing model construction with single tracer and nothing tracer..."
         for arch in archs
-            grid = RectilinearGrid(arch, size=(1, 1, 1), extent=(1, 2, 3))
-
-            model = NonhydrostaticModel(; grid, tracers=:c, buoyancy=nothing)
-            @test model isa NonhydrostaticModel
+            for grid in grids
```

Comment:
grids seem to be on CPU() so putting all these within a loop `for arch in archs ... end` seems to have no effect, right?

Reply:
Thanks for the suggestion.